sources/Directory.Build.props

```diff
@@ -18,7 +18,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
 
   <ItemGroup>
-    <InternalsVisibleTo Include="$(MSBuildProjectName).UnitTests" />
+    <InternalsVisibleTo Include="$(MSBuildProjectName).UnitTests" PublicKey="$(MathSharpPublicKey)" />
```

Comment:
I haven't made this change on my side yet, but it makes it a bit easier to define IVTs for external components, such as if you are doing moqs/etc